web/styles.css

```diff
@@ -9,6 +10,11 @@ body {
     color: #c6c7c7;
     margin: 0;
     overflow-x: hidden;
+    overflow-y: auto;
+}
+
+#container {
+    position: relative;
```

Comment:
Allows children elements to use absolute positioning relative to the container instead of the page